sync-todo/v2/client/flutter/lib/main.dart

```diff
@@ -8,13 +8,14 @@ import 'package:flutter_todo/realm/app_services.dart';
 import 'package:flutter_todo/screens/homepage.dart';
 import 'package:flutter_todo/screens/log_in.dart';
 
+dynamic realmConfig;
```

Comment:
this seems funky. what's the thought for a top scope `dynamic` variable?

Reply:
This seems to be the easiest way to share the realmConfig info (including the Data Explorer URL) between `main()` and `App`. Do you have a suggestion for another way to do it?

Reply:
can you pass it as a variable from main to App?

Reply:
🙃 Of course! lol

I've been context switching too much. Updated to remove the global dynamic.